### ISSUE TITLE
New branch is not tracking the remote

### ISSUE THREAD

**Issue by kornelski** [gjtorikian/no-more-masters#2]
When creating the new main branch, make sure it's tracking the remote (e.g. push with `--set-upstream`)

**Comment by gjtorikian** [comment]
Ah, thanks again! Fixed and released in 0.2.0.

**Comment by kornelski** [comment]
The fix didn't work. It tries to run:

git branch --set-upstream-to origin/main

but that is not a good way to do it:

```
   Error: Command failed with exit code 1: git branch --set-upstream-to origin/main
    error: the requested upstream branch 'origin/main' does not exist
    hint:
    hint: If you are planning on basing your work on an upstream
    hint: branch that already exists at the remote, you may need to
    hint: run "git fetch" to retrieve it.
    hint:
    hint: If you are planning to push out a new local branch that
    hint: will track its remote counterpart, you may want to use
    hint: "git push -u" to set the upstream config as you push.
```


but it should be setting the upstream on `push`. Just push with `-u`, and you won't need the dance of creating untracked branch, fetching, and fixing tracking.


```sh
git push -u origin main:main
```
